src/util/getCaretPositionDetails.ts

```diff
@@ -0,0 +1,60 @@
+interface CaretPositionDetails {
+  focusNode: Node | null,
+  offset: number,
+}
+
+/**
+ * Recursively iterates the nodes children and returns focusNode and offset where the relative offset ends.
+ */
+export const recursiveFocusNodeFinder = (node: Node, relativeOffset: number): CaretPositionDetails | null => {
```

Comment:
`recursiveFocusNodeFinder` shouldn't be exported.

Reply:
Will fix this. Thanks!